h/static/scripts/login-forms/components/SignupForm.tsx

```diff
@@ -23,6 +24,11 @@ type IdProviderBadgeProps = {
 function IdProviderBadge({ provider, identity }: IdProviderBadgeProps) {
   return (
     <div data-testid="id-badge">
+      {provider === 'google' && (
+        <a href="https://myaccount.google.com" target="_blank" rel="noreferrer">
```

Comment:
The Google account management page seemed like a reasonable thing to link to here, though we could also use google.com or not make this a link.

Reply:
I'd probably just omit the link. I think linking to Google's account manager page or to Google.com is just potentially confusing, and not sure it's of much use to the user. We _should_ definitely improve the label when indicating a connected Google account by including the email address and/or display name, and that'll be coming in a future PR.

Reply:
OK, I've removed the link for the moment.